docs/advanced/advanced_troubleshooting.md

```diff
@@ -0,0 +1,124 @@
+# Troubleshooting
+Use this section for advanced troubleshooting information for your CSI driver.
+
+- [Verifying the CSI driver is running](#Verifying-the-CSI-driver-is-running)
+- [Multipath troubleshooting](#Multipath-troubleshooting)
+- [General troubleshooting](#General-troubleshooting)
+
+## Verifying the CSI driver is running
+
+  Verify that the CSI driver is running. (Make sure that all components are in the  _Running_ state), using the `$> kubectl get all -n <namespace> -l csi` command.
+
+## Multipath troubleshooting
+
+Use this information to help pinpoint potential causes for multipath failures.
+
+**Note:** Run these commands should be run on one of the worker nodes.
+
+-   **Display multipath information (FC and iSCSI)**
+
+    Display multipath information, using the `sudo multipath -ll` command.
+
+    <pre>
+    mpathb (3600507680283851530000000000000a6) dm-0 IBM,2145
+    size=1.0G features='1 queue_if_no_path' hwhandler='1 alua' wp=rw
+    |-+- policy='service-time 0' prio=50 status=active
+    | `- 3:0:0:0 sda 8:0  active ready running
+    `-+- policy='service-time 0' prio=10 status=enabled
+    `- 2:0:0:0 sdb 8:16 active ready running
+    
```

Comment:
continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r805321106):
>i think i fixed this now

you can verify changes using `View file`:
![image](https://user-images.githubusercontent.com/57443811/153753875-7785bd98-3e88-4f03-a9b7-3dbd44e791f4.png)